cmd/process-agent/collector.go

```diff
@@ -176,7 +176,18 @@ func (l *Collector) runCheck(c checks.Check, results *api.WeightedQueue) {
 	} else {
 		checks.StoreCheckOutput(c.Name(), nil)
 	}
-	l.messagesToResults(start, c.Name(), messages, results)
+
+	if c.Name() == config.PodCheckName {
+		// Pod check returns a list of messages can be divided into two parts : pod payloads and manifest payloads
```

Comment:
of messages, that...: pod and manifest payloads

Reply:
Seeing this, I personally prefer putting them into a dedicated handler function where you can move the comment above to it i.e.

```
// By default ...
handlePodChecks(...){
 l.messageToResult...
 if ...
    l.messageToResult...
}
```

Reply:
I add a a handler https://github.com/DataDog/datadog-agent/blob/d624a9933d3f6122d59ba768ffbc5b78b6fa4744/cmd/process-agent/collector.go#L746-L750